addons/addon-base-raas/packages/base-raas-services/lib/study/study-service.js

```diff
@@ -526,6 +527,21 @@ class StudyService extends Service {
     return result;
   }
 
+  /**
+   * IMPORTANT: Do NOT call this method directly from a controller, this is because
+   * this method does not do any authorization check.  It will return the study given
+   * a study id no matter who the requestContext principal is.
+   */
+  async listStudiesForAccount(requestContext, { accountId }) {
+    const result = await this._query()
+      .index(this.accountIdIndex)
+      .key('accountId', accountId)
+      .limit(1000)
+      .query();
+
+    return result;
```

Comment:
We are missing the logic to convert dbEntities to just studyEntities.
So, you want to do something like
`return _.map(result, toStudyEntity);`
Look for an example in the same service.

Reply:
Sure, but only the `id` field is used from the list returned from this method.

Reply:
For the use of 'toStudyEntity', this is not related to how many properties you need.  This is more related to that fact that dbEntities are never exposed to any methods,  because sometime dbEntities will include properties that are purely for database storage, such as composite keys with prefixes like 'ACT#', etc.

Aside from the need to use toStudyEntity, do you only need the id or you also need the bucket and region information stored in the study entity?

Reply:
Done